addon/services/google-maps-api.js

```diff
@@ -71,19 +71,57 @@ Learn more: https://ember-google-maps.sandydoo.me/docs/getting-started`,
     );
 
     return new Promise((resolve, reject) => {
-      window.initGoogleMap = bind(() => {
+      const params = new URLSearchParams(new URL(src).search);
```

Comment:
1. I would use the `config` directly at this point. I don't see a need to use URI-escaped values here.
2. As an option, we could also do something `JSON.stringify(Object.fromEntries([["version", "whatever"], ["emptyOption", ""]].filter((_, x) => x)))`. Or push to an array of options, then  `join(',')`. I don't really have a preference here, just worried about forgetting a comma somewhere.
3. Are there versions of google maps that are still available, but don't support this? Should there be a version check to fall back to loading from `src`?